src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -19,24 +19,9 @@ export const getters = {
     },
     regionOptions: (state: FilteredDataState, getters: any, rootState: RootState, rootGetters: any) => {
         const shape = rootState.baseline && rootState.baseline.shape ? rootState.baseline.shape : null;
-        let result =  shape && shape.filters && shape.filters.regions ? shape.filters.regions : [];
-        //TODO: This is just to return something here until we have actual filter data from the API
-        if (result && result.length == 0){
-            result = [
-                {name: "Northern Region", id: "MWI.1", options: [
-                        {name: "Chitipa", id: "MWI.1.1"},
-                        {name: "Karonga", id: "MWI.1.2"}
-                    ]},
-                {name: "Central Region", id: "MWI.2", options: [
-                        {name: "Dedza", id: "MWI.2.1"},
-                        {name: "Dowa", id: "MWI.2.2"}
-                    ]},
-                {name: "Southern Region", id: "MWI.3", options: [
-                        {name: "Balaka", id: "MWI.3.1"}
-                    ]}
-            ];
-        }
-        return result;
+        return shape && shape.filters &&
+                        shape.filters.regions &&
+                        (shape.filters.regions as any).options ? (shape.filters.regions as any).options : [];
```

Comment:
These filters have an additional level 'options' now to get to the top level, however I haven't regenerated types from the latest hintr yet as assume will have same problem with recursion as last time...